package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Compile LightScript to JavaScript.",
   "main": "index.js",
   "scripts": {
-    "test": "node node_modules/.bin/mocha test",
+    "test": "mocha test",
```

Comment:
Yeah, see the [docs on `npm run`](https://docs.npmjs.com/cli/run-script), specifically:

> In addition to the shell's pre-existing `PATH`, `npm run` adds `node_modules/.bin` to the `PATH` provided to scripts. Any binaries provided by locally-installed dependencies can be used without the `node_modules/.bin` prefix.